docs/rst/tutorials/core/discoveryserver/discoveryserver_complex.rst

```diff
@@ -0,0 +1,380 @@
+This method combines the flexibility of UDP-based discovery with the dependability of TCP or SHM for large data transfers.
+
+*Large Data mode* can be easily enabled using the ``FASTDDS_BUILTIN_TRANSPORTS`` environment variable, XML profiles, or code:
+
+.. code:: bash
+
+    export FASTDDS_BUILTIN_TRANSPORTS=LARGE_DATA
+
+This simplifies the configuration process and helps optimize large data transmission without needing extensive network adjustments.
+
+If further adjustments are necessary, it is important to assess the specific requirements of your architecture to ensure optimal performance.
+Fast DDS provides various tuning options to prevent issues such as buffer overflows and retransmissions when using UDP.
+Key configurations include increasing socket buffer sizes, adjusting the transmit queue length, and employing flow controllers to manage data transmission rates.
+
+For more details and advanced configurations, refer to the official documentation `here <https://fast-dds.docs.eprosima.com/en/latest/fastdds/use_cases/large_data/large_data.html>`__.
+
+.. note::
+
+    For compatibility issues, Discovery Server has to be configured with TCP transport protocol when Large Data is set.
+
+Prerequisites
+-------------
+
+It is required to have previously installed Vulcanexus using one of the following installation methods:
+
+* `Linux binary installation <https://docs.vulcanexus.org/en/latest/rst/installation/linux_binary_installation.html>`__
+* `Linux installation from sources <https://docs.vulcanexus.org/en/latest/rst/installation/linux_source_installation.html>`__
+* `Docker installation <https://docs.vulcanexus.org/en/latest/rst/installation/docker.html>`__
+
+Additionally, you need to have OpenCV installed.
+You can follow the `OpenCV documentation
+<https://docs.opencv.org/2.4/doc/tutorials/introduction/table_of_content_introduction/table_of_content_introduction.html#table-of-content-introduction>`__
+for detailed installation instructions.
+
+You will also need the ROS package ``image_tools``, which can be installed by running:
+
+.. code:: bash
+
+    sudo apt-get install ros-humble-image-tools
+
+Lastly, ensure you have a working webcam connected to both of the hosts.
+
+Considerations for Video Streaming
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Before proceeding with the tutorial, it's important to consider the following factors when streaming large video images over Wifi:
+
+1. Network Bandwidth:
+    Ensure that the Wifi network has sufficient bandwidth to support the video stream's bitrate, especially when multiple devices are connected.
+    Higher-resolution or higher-frame-rate video streams demand more bandwidth, so consider compressing the video or reducing the resolution if the Wifi network has limitations.
```

Comment:
I'd add something like "stay tuned for upcoming tutorials on how to do this in ROS 2"

Reply:
Let's talk about this section tomorrow as it seems too passive-aggressive for me. I'd give some hints on how to solve this as changing the system sockets to higher values to manage higher data rates, or use netstat or iptraf to see the network conditions.